code/modules/clothing/suits/armor.dm

```diff
@@ -184,9 +187,25 @@
 			return
 		H.forceMove(picked)
 		H.rad_act(rad_amount)
+		active = 0
+		src.icon_state = "reactiveoff"
+		src.item_state = "reactiveoff"
+		cooldown++
```

Comment:
Instead of increasing the cooldown var, set it to `world.time + 100` and check if `world.time < cooldown` on line 140.
This basically replaces lines 193-197 with one line.
Also you forgot to define the cooldown var